schemas/openshift/cluster-1.yml

```diff
@@ -196,6 +196,10 @@ properties:
         properties:
           soakDays:
             type: integer
+          locks:
+            type: array
+            items:
+              type: string
```

Comment:
I would really like for this to be an enumeration so that it's more difficult to make a mistake.  But we don't like enums, so maybe a setting?

Reply:
https://issues.redhat.com/browse/APPSRE-4709